plugins/UM3NetworkPrinting/resources/qml/ExpandableCard.qml

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) 2018 Ultimaker B.V.
+// Cura is released under the terms of the LGPLv3 or higher.
+
+import QtQuick 2.2
+import QtQuick.Controls 2.0
+import UM 1.3 as UM
+import Cura 1.0 as Cura
+
+// TODO: Theme & documentation!
+// The expandable component has 3 major sub components:
+//      * The headerItem Always visible and should hold some info about what happens if the component is expanded
+//      * The popupItem The content that needs to be shown if the component is expanded.
+Item
+{
+    id: base
+
+    property var expanded: false
+    property var borderWidth: 1
+    property color borderColor: "#EAEAEC"
+    property color headerBackgroundColor: "white"
+    property color headerHoverColor: "#f5f5f5"
+    property color drawerBackgroundColor: "white"
+    property alias headerItem: header.children
```

Comment:
Isn't it headerItems if you're setting the children (since it can be a list). Item implies that it can be only a single one.

Reply:
I thought it could be only one (since you'd write `headerItem: Row { ... }` or whatever), and `children` was just the QML nomenclature for that?